src/main/java/org/bigredbands/mb/views/MainView.java

```diff
@@ -76,9 +76,11 @@ private void initialize() {
 
     /**
      * Create the project view and display it
+     * 
+     * @param showWizard - true if the user is loading the project, false if this is a new project
```

Comment:
```suggestion
     * @param showWizard - whether the Song Constants wizard should be shown when creating the project view
```

The phrasing of the function documentation, like the variable name, should avoid referencing business logic (and, as it is now, the description is backwards as far as I can tell). The same should be updated in `ViewInterface.java`.

Reply:
Oh right, thank you for the catch